### PR TITLE
Added 8BitDo Ultimate 2.4g / Wired for PC Controllers supported

### PR DESCRIPTION
**Pull request by 8BitDo**

### CODE REVIEW

File: udev/8BitDo_Ultimate_Wired_Controller.cfg [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# 8BitDo Ultimate Wired Controller        - https://www.8bitdo.com/        - https://www.8bitdo.com/ultimate-wired-controller/
+# Firmware v1.01        - https://support.8bitdo.com/        - https://support.8bitdo.com/firmware-updater.html
+# This is with the device started in Android (D-Input) mode.
+
+input_driver = "udev"
+input_device = "8BitDo 8BitDo Ultimate Wired Controller"
```
Comment (hrzhu) [inline udev/8BitDo_Ultimate_Wired_Controller.cfg:6]: Two 8Bitdo?

File: udev/8BitDo_Ultimate_wireless_Controller.cfg [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# 8BitDo Ultimate 2.4 Wireless Controller        - https://www.8bitdo.com/        - https://www.8bitdo.com/ultimate-2.4g-wireless-controller/
+# Firmware v1.08        - https://support.8bitdo.com/        - https://support.8bitdo.com/firmware-updater.html
+# This is with the device started in Android (D-Input) mode.
+
+input_driver = "udev"
+input_device = "8BitDoUltimate wireless Controller for PC"
```
Comment (hrzhu) [inline udev/8BitDo_Ultimate_wireless_Controller.cfg:6]: Possible missing whitespace between 8BitDoUltimate

**Comment by hrzhu** [comment]
@8BitDo No Ultimate Bluetooth controller configs?

File: udev/8BitDo_Ultimate_Wired_Controller.cfg (hunk 1 above)
Comment (offalynne) [reply udev/8BitDo_Ultimate_Wired_Controller.cfg:6]: On what basis did you leave this review comment, exactly ? The vendor submitted these configurations themself and have obviously been thorough. Look at other entries, it is very common vendor name is in duplicate.

File: udev/8BitDo_Ultimate_wireless_Controller.cfg (hunk 2 above)
Comment (offalynne) [reply udev/8BitDo_Ultimate_wireless_Controller.cfg:6]: @hrzhu Possible ? Have you tested your assumption ?

**Comment by offalynne** [comment]
> @8BitDo No Ultimate Bluetooth controller configs?

@hrzhu Have you tried the Ultimate Bluetooth model with RetroArch yourself ? I assume not. In Bluetooth mode it spoofs a Switch Pro controller. In 2.4g, the dongle spoofs an Xbox controller on Windows only. No configuration files are necessary for the controller to operate as intended in RetroArch.

**Pull request merged**